src/components/ContentWrapper.jsx

```diff
@@ -62,10 +61,44 @@ const ContentWrapper = (props) => {
   const user = useSelector((state) => state.user.current);
 
   const samples = useSelector((state) => state.samples);
-
   // selectedExperimentID holds the value in redux of the selected experiment
   // after loading a page it is determined whether to use that ID or the ID in the route URL
   // i.e. when we are in data management there is not exp ID in the URL so we get it from redux
+  const getCurrentStatusScreen = (isSeurat, backendStatus) => {
```

Comment:
this function is pretty confusing to read with so many different and nested if's. Could you try to refactor it and add comments to make it more clear? One possible way would be to get rid of the nested ifs and add comments to make all cases clear like:
```
// Seurat cases

// Seurat Running Error
  if (isSeurat && seuratRunningError) {
    const seuratErrorMessage = pipelineErrorUserMessages[backendStatus?.seurat?.error?.error];
    return { type: 'seurat', message: seuratErrorMessage, status: 'error' };
  }
// Normal seurat running status
  if (seurat && seuratRunning) {
    return { type: 'seurat', status: 'running', completedSteps: completedSeuratSteps };
  }

// Normal pipelines
// Gems 2 Error
 if (!seurat && gem2sRunningError) {
        return { type: 'ge
```

I don't love this approach so I'm open to suggestions. The only goal is to make reading this function or adding / removing cases easier because it's confusing as it is.